andela/titan-dev-ah-frontend
# #163518645 Social Auth

#### What does this PR do?
Add social auth
#### Description of Task to be completed?
- Activated social login button
- A user can log in with Google, Facebook, or Twitter
#### How should this be manually tested?
- clone the repo and check out the following branch `ft-social-auth-163518645 `
- Also, clone the backed repo and check out branch `ch-server-message-165307938`
- run `yarn start` and navigate to /sign_in in your browser
- Try one of the social auth buttons
### Related Pivotal tracker ID
#163518645
